membership-attribute-service/conf/PROD.public.conf

```diff
@@ -11,20 +11,13 @@ logger.application=INFO
 touchpoint.backend.default=PROD
 touchpoint.backend.test=UAT
 
+play.filters.cors.supportsCredentials = true
```

Comment:
This might be defaulted to true, but to save confusion, I thought it should be hardcoded anyway.